internal/server/authn/oidc.go

```diff
@@ -132,10 +122,10 @@ func (o *oidcImplementation) ExchangeAuthCodeForProviderTokens(code string) (raw
 	}
 
 	if err := idToken.Claims(&claims); err != nil {
-		return "", "", time.Time{}, "", fmt.Errorf("id cliams: %w", err)
+		return "", "", time.Time{}, "", fmt.Errorf("id claims: %w", err)
 	}
 
-	return rawAccessToken, rawRefreshToken, exp, claims.Email, nil
+	return rawAccessToken, rawRefreshToken, exchanged.Expiry, claims.Email, nil
```

Comment:
Some IDPs do not return the access token as a JWT, so we can rely on this response expiry instead of checking the actual JWT instead